src/cms/blocks.md

```diff
@@ -4,7 +4,9 @@ title: Blocks
 
 A block is a modular unit of content that can be positioned most anywhere on the page. Content blocks are sometimes referred to as static blocks, or CMS blocks. They can be used to display fixed information such as text, images, and embedded video, as well as dynamic information from a widget or that originates in a database or other source. Most elements on the home page are blocks that can be easily managed.
 
-You can create custom blocks of content without writing any code, and assign them to appear in a specific place in the page layout. Blocks can also be defined and positioned by making a [layout update]({% link design/layout-updates.md %}) in XML code.
+You can create custom blocks of content without writing any code, and assign them to appear in a specific place in the page layout. Blocks can be positioned with a [Widget]({% link cms/widget-static-block.md %}) or by composing a [layout update]({% link design/layout-update-place-block.md %}) in XML and saving it on the server.  To learn more, see the [Layout][1] section in the developer documentation.
```

Comment:
Do you mean inside a widget?

Reply:
No, "with" a widget is accurate.  I could change it to "with the Widget tool..."

Reply:
Fixed.